quinn-proto/src/connection/mod.rs

```diff
@@ -1175,6 +1216,7 @@ impl Connection {
                     self.spaces[space].pending_acks.subtract_below(acked);
                 }
                 ack_eliciting_acked |= info.ack_eliciting;
+                self.path.mtud.on_acked(space, packet, info.size);
```

Comment:
Any considerations for moving this into `self.on_packet_acked()` instead of keeping it outside?

Reply:
Moving the call to `mtud.on_acked` into `self.on_packet_acked` seems to make sense, but requires modifying the signature of the latter, so it has access to the packet number (which it currently lacks). I'll do it ;)

Reply:
That might not be an improvement. I was just going off of, are there other callers of `on_packet_acked()`, how do these things conceptually relate? But it looks like this is the only caller that potentially uses the `Data` space, so it's probably fine to leave as is.

Reply:
Ok, just reverted the change